src/app/partnerzy/_components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import Image from "next/image";
+import useEmblaCarousel from "embla-carousel-react";
+import Autoplay from "embla-carousel-autoplay";
+
+interface Partner {
+  id: string;
+  name: string;
+  href: string;
+  image: string; // Optional?
+}
+
+interface Props {
+  items: Partner[];
+}
+
+// Todo: Adjust spacings for viewports. Maybe extract <CarouselItem /> ?
```

Comment:
Do we want Carousel to be an independent generic component for usage with other items or can it remain Partner specific?

Reply:
https://github.com/Delicadency/AK1944/pull/72/#issuecomment-2735767751

Reply:
[Reply here](https://github.com/Delicadency/AK1944/pull/72/#issuecomment-2735767751)